src/dataflow/components/dataflow-program-graph.tsx

```diff
@@ -235,6 +257,22 @@ export class DataflowProgramGraph extends React.Component<IProps, IState> {
           boxWidth: 6,
           fontSize: 12,
           fontFamily: "'Ubuntu', 'Arial', sans-serif"
+        },
+        // TODO: this approach detects the legend click and then keeps
+        // track of which dataset sequence is shown/hidden in the component state.
+        // to prevents error, we must also stop thge subsequent render in shouldComponentUpdate.
+        // in the future, we should investigate an alternative approach that doesn't require
+        // keeping track of chart.js state in our component state.
+        // tslint:disable-next-line
+        onClick: function(e, legendItem) {
```

Comment:
```suggestion
        onClick(e, legendItem) {
```
I think this will allow you to eliminate the `tslint:disable-next-line`

Reply:
That works!  Thanks!